plugins/cc-spex/skills/cc-spex-init/scripts/bootstrap.sh

```
@@ -0,0 +1,82 @@
+    *) echo "ERROR: unknown option: $1" >&2; usage >&2; exit 2 ;;
+  esac
+done
+
+case "$integration" in codex|claude|opencode|auto) ;; *) echo "ERROR: invalid integration" >&2; exit 2 ;; esac
+case "$security" in safe|autonomous|yolo|interactive) ;; *) echo "ERROR: invalid security" >&2; exit 2 ;; esac
+case "$extensions" in
+  recommended|all|interactive) ;;
+  *)
+    old_ifs=$IFS
+    IFS=,
+    set -- $extensions
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)